worker/src/main/scala/org/apache/celeborn/service/deploy/worker/PushDataHandler.scala

```diff
@@ -133,13 +131,13 @@ class PushDataHandler extends BaseMessageHandler with Logging {
     val isMaster = mode == PartitionLocation.Mode.MASTER
 
     // For test
-    if (isMaster && testPushMasterDataTimeout && !pushMasterDataTimeoutTested) {
-      pushMasterDataTimeoutTested = true
+    if (isMaster && testPushMasterDataTimeout && !PushDataHandler.pushMasterDataTimeoutTested.getAndSet(
+        true)) {
       return
     }
 
-    if (!isMaster && testPushSlaveDataTimeout && !pushSlaveDataTimeoutTested) {
-      pushSlaveDataTimeoutTested = true
+    if (!isMaster && testPushSlaveDataTimeout && !PushDataHandler.pushSlaveDataTimeoutTested.getAndSet(
+        true)) {
```

Comment:
```suggestion
    if (!isMaster && testPushSlaveDataTimeout && 
        !PushDataHandler.pushSlaveDataTimeoutTested.getAndSet(true)) {
```

Reply:
`spotless` reformat this to 2 spaces instead 4.